src/diagonal.works/b6/api/functions/features.go

```diff
@@ -91,14 +92,15 @@ func (a *arrayAreaFeatureCollection) Next() (bool, error) {
 
 var _ api.Collection = &arrayAreaFeatureCollection{}
 var _ api.Countable = &arrayAreaFeatureCollection{}
+*/
 
-func findAreasContainingPoints(context *api.Context, points api.PointFeatureCollection, q b6.Query) (api.AreaFeatureCollection, error) {
+func findAreasContainingPoints(context *api.Context, points b6.Collection[any, b6.Point], q b6.Query) (b6.Collection[b6.FeatureID, b6.AreaFeature], error) {
```

Comment:
y is this any / not featureid ?

Reply:
Because the function actually doesn't care about the key, it just uses the values, so this way Collection[int, b6.Point] will work for example, which is what you get when converting a list of Points in Python into a collection.